File: src/Test/PolymorphismFFICheatcode.go
package Test_PolymorphismFFICheatcode


type Showable_cheatcode interface {
	Show() string
}
type MyInt_cheatcode int
func (m MyInt_cheatcode) Show() string { return "Int" }

func RunPolymorphismFFICheatcode(limit float64) float64 {
	n := int(limit)
	count := 0
	var s Showable_cheatcode = MyInt_cheatcode(0)
	for i := 0; i < n; i++ {
		if s.Show() == "Int" {
			count++
		}
	}
	return float64(count)
}

File: src/Test/PolymorphismFFICheatcode.rs
pub fn Test_PolymorphismFFICheatcode_runPolymorphismFFICheatcode(mut limit: i64) -> i64 {
    let a = vec![vec![1, 2], vec![3, 4]];
    let b = vec![vec![1, 2], vec![3, 4]];
    let mut count = 0;
    for _ in 0..limit {
        if a == b { count += 1; }
    }
    count
}